### ISSUE TITLE
device_tracker.icloud3_lost_iphone: 'Icloud3' object has no attribute '_display_status_info_msg'

### ISSUE THREAD

**Issue by Mic92** [gcobb321/icloud3#87]
When using the icloud3_lost_iphone service I get the following error with home assistant 0.116.4:

```
Failed to call service device_tracker/icloud3_lost_iphone. 'Icloud3' object has no attribute '_display_status_info_msg'	
```

This is the full stack trace:

```
Oct 24 15:06:00 eve hass[13202]: 2020-10-24 15:06:00 WARNING (SyncWorker_1) [custom_components.icloud3.device_tracker] Lost Phone Alert Error: Alerts can only be sent when using tracking_method FamShr
Oct 24 15:06:00 eve hass[13202]: 2020-10-24 15:06:00 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection.140511319553936] 'Icloud3' object has no attribute '_display_status_info_msg'
Oct 24 15:06:00 eve hass[13202]: Traceback (most recent call last):
Oct 24 15:06:00 eve hass[13202]:   File "/nix/store/0cyykck01zhpcdaziry4vhd755nhlzib-homeassistant-0.116.4/lib/python3.8/site-packages/homeassistant/components/websocket_api/commands.py", line 138, in handle_call_service
Oct 24 15:06:00 eve hass[13202]:     await hass.services.async_call(
Oct 24 15:06:00 eve hass[13202]:   File "/nix/store/0cyykck01zhpcdaziry4vhd755nhlzib-homeassistant-0.116.4/lib/python3.8/site-packages/homeassistant/core.py", line 1335, in async_call
Oct 24 15:06:00 eve hass[13202]:     task.result()
Oct 24 15:06:00 eve hass[13202]:   File "/nix/store/0cyykck01zhpcdaziry4vhd755nhlzib-homeassistant-0.116.4/lib/python3.8/site-packages/homeassistant/core.py", line 1374, in _execute_service
Oct 24 15:06:00 eve hass[13202]:     await self._hass.async_add_executor_job(handler.func, service_call)
Oct 24 15:06:00 eve hass[13202]:   File "/nix/store/346skv0d24rqnf4npknbp9h5bs14j8zy-python3-3.8.6/lib/python3.8/concurrent/futures/thread.py", line 57, in run
Oct 24 15:06:00 eve hass[13202]:     result = self.fn(*self.args, **self.kwargs)
Oct 24 15:06:00 eve hass[13202]:   File "/var/lib/hass/custom_components/icloud3/device_tracker.py", line 899, in _service_callback_lost_iphone
Oct 24 15:06:00 eve hass[13202]:     ICLOUD3_GROUP_OBJS[group]._service_handler_lost_iphone(
Oct 24 15:06:00 eve hass[13202]:   File "/var/lib/hass/custom_components/icloud3/device_tracker.py", line 8773, in _service_handler_lost_iphone
Oct 24 15:06:00 eve hass[13202]:     self._display_status_info_msg(arg_devicename, log_msg)
Oct 24 15:06:00 eve hass[13202]: AttributeError: 'Icloud3' object has no attribute '_display_status_info_msg'
```

**Comment by Mic92** [comment]
cc @slekwati

**Comment by gcobb321** [comment]
This has been fixed in the v2.2.1c Development version [here}(https://github.com/gcobb321/icloud3/tree/master/development%20area%20-%20v2.2.1).

Download the zip file, unzip it into the icloud3 directory and restart HA.

**Comment by Mic92** [comment]
Thanks!
